diff --git a/dist/index.js b/dist/index.js
index ba566d95956789a83721f4b36796325c603a7923..23659a00e19348f169eae505d12c930171d4bde4 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -175,7 +175,10 @@ export default function netlifyIntegration(integrationConfig) {
 					request,
 					params: {}
 				});
-				ctx.locals = { netlify: { context } }
+				if (typeof ctx.locals !== 'object') {
+					ctx.locals = { netlify: {} };
+				}
+				ctx.locals.netlify = { context }; 
 				// https://docs.netlify.com/edge-functions/api/#return-a-rewrite
 				ctx.rewrite = (target) => {
 					if(target instanceof Request) {
@@ -196,12 +199,14 @@ export default function netlifyIntegration(integrationConfig) {
 					return new URL(target, request.url);
 				};
 				const next = () => {
+					console.log('netlify-edge-next-ctx', ctx.locals);
+					console.log('netlify-edge-next-context', context.locals);
 					const { netlify, ...otherLocals } = ctx.locals;
 					request.headers.set("x-astro-locals", trySerializeLocals(otherLocals));
 					request.headers.set("x-astro-middleware-secret", "${middlewareSecret}");
 					return context.next();
 				};
-
+				console.log('netlify-execute-custom-middleware', ctx.locals)
 				return onRequest(ctx, next);
 			}
 
